docs/integrations/persisting-store-data.md

```diff
@@ -571,3 +571,34 @@ export const useBearStore = create<MyState>()(
   )
 )
 ```
+
+### How do I use deal with Map/Set with the persist api
+
+With the previous persist API, you would use `serialize`/`deserialize` to deal with Map and Set and convert them into Array so they could be parsed into proper JSON.
+The new persist API has deprecated `serialize`/`deserialize`.
+
+Now, you will need to use the `storage` prop, let's say your state use a `Map` to handle a list of `transactions`, then you can convert the Map into an Array in the storage prop:
+
+```typecript
```

Comment:
There is a typo in `typecript`. Please fix it either to `typescript`, or better yet, simply use `ts`.